### DISCUSSION TITLE
M.I.B 3.6.0 - QR Edition

### DISCUSSION THREAD

**Discussion by harman-f** [Mr-MIBonk/M.I.B._More-Incredible-Bash#196] [Show and tell]
Looking for some feedback on new M.I.B features before transfer to a full release :)

V3.6.0 will bring some new and improved stuff!

You can grab it directly from repo: https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/archive/refs/heads/main.zip 

### **New:**
- Custom images can be displayed on screen from GEM or script.
- This is utilized in two places right now:
   1. Infoscreen containing QR codes to the most important MIB ressources
![grafik](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/98130152/a20a7d29-57e2-4c9b-a282-a1e27bcbe657)
![image](https://user-images.githubusercontent.com/98130152/236244886-4eb464ca-b82a-48ef-ae15-6c417ba17d04.png)
   3. A very green confirmation screen after the patching was successful 
![image](https://user-images.githubusercontent.com/98130152/236242442-d4a5dd3a-dfb1-4f2b-a91b-ceb222c04eed.png)
- New EU conversions scripts and logic
   - Additional script to fix conversion issues for screen/skin ID
![grafik](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/98130152/7b7e1f89-d88a-4591-a3e4-d3917b5241e3)
![grafik](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/98130152/bb6e9e60-9e88-4b09-a0b6-71fec9d646e6)
- Log files with time stamps
- Legay FW support: Basic support for very old FW, which do not come with persistence binary. Basic stuff like backup should work.
- New GEM settings: Summer time also, loudness
- TMC menu to load custom TMC configs to unit
- New Skin for VWG13:
  -  https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/VWG13-VW-MIB2.5-Image-MOD
- And more stuff...

### **Improved:**
- Patch process has a ton of new debug outputs, including actions to fix issues.
  - Can identify if unit is already patched with known patch
  - Can report missing patch folder on SD
  - Can identify unknown patch on unit
- Each flash is validated:
  - Unit will not reboot if the flash failed or was incomplete
  - Instructions on how to act are provided on screen
- Some more small things here and there
- Improved EU conversion scripts and GEM menu
- Image MOD for SKG13
   - MOD seize reduced from ~100MB to ~10MB
   - New GEM Menu with QR code to display MOD result
   - https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/SKG13---Skoda-MIB2.5-Image-MOD

--> Check flash script for details: https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/blob/main/apps/flash
 
**Please report back any issues!**

**Comment by fhdoma** [comment]
![20230519_151320](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/38072288/2afa32d5-5927-4ec1-8bec-c29d435fe9d3)
I try new mib I have correct fw and octavia 3 but error with skin change

![20230519_151453](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/38072288/727ac27c-1e9f-40df-a483-c068ee239384)
Info

**Reply by harman-f** [reply]
try latest version again.
This is fixed.

Also check here:
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/SKG13---Skoda-MIB2.5-Image-MOD

**Reply by fhdoma** [reply]
@harman-f I download and install new app and still skin error 
![20230520_134130](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/38072288/be5fbcf9-663e-48f6-9ca7-05d8e1a947d0)

**Reply by harman-f** [reply]
There was a new bug this morning I fixed already. Try latest from now.

**Reply by fhdoma** [reply]
@harman-f  I try download mail. Zip and install and sane issue..

**Reply by harman-f** [reply]
OK, found one more ;)

**Reply by fhdoma** [reply]
@harman-f can download main zip and test? I have original canton and not found any improvement... Was by possible add to navigation enable 3d building and more config from nav hidden for driver Configuration?

**Reply by fhdoma** [reply]
I try last main skin script komplete done restart mib but skin is old...

**Reply by fhdoma** [reply]
@harman-f script done successfully but after restart is same template

**Reply by harman-f** [reply]
You have to switch to skin 3

WIll add that in the next step to also change coding

**Reply by fhdoma** [reply]
@harman-f  I change skin to 3 save and restart and navigation cannot boot :( I have onli Carista and Carscanner cannot go to green menu 
![20230522_145334](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/38072288/c504e315-391f-4bf6-ae11-56ce02fe92bc)

**Reply by fhdoma** [reply]
@harman-f have way how can revert back to skin 1? Mib does load. Have tool car scanner or carista

**Reply by harman-f** [reply]
That is odd....

Skin is Long coding byte 17

**Comment by fhdoma** [comment]
I change long code 17 to 01 and 18 to 00 and work old style @harman-f ...  But now have on diagnostic issue 
![Screenshot_20230522_170934](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/38072288/5a93f91c-381c-4267-8014-c38b9b4e7d81)
![Screenshot_20230522_171143_Carista](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/38072288/f2e72b80-21d5-4edf-8fad-2b0313d66033)

**Reply by harman-f** [reply]
This is SVM, you get it after every FW install (also M.I.B install)

There is a "SVM fix" function within M.I.B.

It also turned out, that the unit did not like the way skin files were replaced on unit. We spent quite a bit more time on debugging and latest commit to repo is working now!

Btw: You might need to reinstall app.img via FW update. Are any images missing now, after the patch failed?

**Comment by fhdoma** [comment]
@harman-f hello, thank you. I download last update main branch, install, then install skin 3 mib restart skin is old. What is next step?

Image old skin are OK

**Comment by fhdoma** [comment]
Work

![20230523_075015](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/38072288/a5d827de-df6d-4334-a418-4bf56dc20de2)
![20230523_075011](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/38072288/5995f420-69ee-4f2a-9dc3-afd12c367962)

**Comment by fhdoma** [comment]
This skin enable 3d building in nav.. Will be possible as feature in navigation without application this skin?

**Reply by harman-f** [reply]
Can you share a screenshot of 3D buildings?

**Comment by Gintazas** [comment]
![IMG_20230526_205253](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/48887740/1936e0fd-b8e5-4da9-92cd-e6da254743c1)
Hi. i am new to update device. Unlock GEM and have a question. Can i install MIB 3.6 on my unite .MHS2_ER_AU_P1242 A4B9 2017? when go to install isnt Laucher Y and "start update" is not actyve. 
![IMG_20230526_123543](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/48887740/d92c32fc-8b7f-4b48-8dd7-eb261767c973)
How it can be that before SW update GEM was 4.5s, after update got down to 4.12s?
![IMG_20230524_113103](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/48887740/46d8c8fa-e6db-4768-a8d3-96ed8785b7d1)
![IMG_20230525_194456](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/48887740/5f243e71-e063-4b1c-b88b-f28a7f1eb4f8)

**Reply by harman-f** [reply]
Wrong FW Type. 
M.I.B only supports MHI2

**Comment by fhdoma** [comment]
I use this version on mib 2.5 and no issue, was by fine some more customization navigation or car but for this are another obd2 tools

**Comment by fhdoma** [comment]
@harman-f hello I try last version, I see more AID images for Skoda, I reinstall skin but is some changed? I see all same

**Comment by fhdoma** [comment]
@harman-f hello can you for skoda add transparent background for radio station https://github.com/jilleb/mib2-toolbox/wiki/Graphics-modification:-Transparent-background-for-radiostation-logos-%5BSeat---Skoda%5D?

**Reply by harman-f** [reply]
if you create a fitting file, I can add that option.

**Reply by fhdoma** [reply]
@harman-f  I don't know how

**Comment by harman-f** [comment]
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/releases/tag/V3.6.0

**Comment by R3DcooDe** [comment]
Hello Mate how are you
with this new update to EU Conversation menu im a little confused - please can you tell me right way to do it 

so i installed mib opened and go to eu_conversion menu - , i click EU Conversion - i wait for mib to do the magic  //   after unit reboot what to do here im confused // in older mib there was some other steps after this one -  " eu conversion coding "  with new update is missing :/   2) in the new update fix skin/ screen bug  is listed as step 3) so my question in, do i need always run this command after running EU_Conversion //    and there it says do not reboot unit directly install EU Firmware --  this lead to my other concern WHat about formatting NAV database,   when to format nav database /  after runing EU_Conversion / or after installing EU Firmware  /    i really appreciate if you help me with Command Right Order to use it to convert to European firmware /  for example 1) advaced settings EU Conversion , 2) skin patch , 3 format database , 4 install european fimware ,,     thank you

**Reply by harman-f** [reply]
This is currently on the flow, with many basic changes.

Get latest M.I.B directly from repo.

It is down to a one button solution, with better instructions. Give that a shot and repoprt back.

**Reply by R3DcooDe** [reply]
aa okay so when i press first EU_Conversion   program will automatically do all the steps including formatting NAV Database .  
also do i need to use fix skin/screen bug - every time for evry unit i convert or it is only in a particular vehicles for example VW 

thank you

**Comment by neoalius** [comment]
Tested enabling the custom TCM in australia and lost the existing TMCPro too.
![Audi_A4_TMCPro](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/131591423/f34c41af-8816-420f-a1e4-7139634e481f)

Logs attached ...
[MU1438-JTB-07428.05.16119300F9 (1).log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11711183/MU1438-JTB-07428.05.16119300F9.1.log)

**Reply by harman-f** [reply]
Something seems to be wrong with the folders.
Will have a look tonight and come back to you.

**Reply by harman-f** [reply]
@neoalius

Get latest code from repo.

clean /logs/ folder and run again.
It will most likely still not work, but should help to find the issue!

Please share your backup and log files (filehorst.de)

**Reply by neoalius** [reply]
Apologies for the delay,

Please find the logs after repeating the activation..
Also to mention i do get the Traffic Messages but there is no overlay in maps, or delay included in the travel time due to those obstructions.

[MU1438-JTB-07428.05.16119300F9.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11858381/MU1438-JTB-07428.05.16119300F9.log)

Uploaded Backup to :
https://drive.google.com/drive/folders/1kbSgd0tpeaY1W4PFul8jztcjYkPKdu53?usp=sharing

**Comment by harman-f** [comment]
continue here:
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/276

**Comment by elocc** [comment]
![IMG_20231125_153646](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/141194535/c0fa2fdb-5e62-4e8d-8f71-d50ac7d5c860)
After successful install Mib, and patching, carplay and aa not working?
Usb cable showing only changing

**Comment by vanitoso** [comment]
hi, I installed mib and everything is ok but I can't install the vw alpine mod, it gives me an error and goes back to the stock skin, try to install skin 5 but nothing, still the same error, I have mib2high MHI2_ER_VWG13_P4521 MU1367, in my firmware only 3 skins, 0 basic, 1 current and 4 Arabic

**Comment by emredurgac** [comment]
![20240619_190934000_iOS](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/173287161/8401498c-52e3-4225-8997-6a52f199b1a2)
![20240619_190941000_iOS](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/173287161/fb4456ba-5422-4068-b811-9305cc378ae9)
Hallo bei mir funktioniert es nicht den Skin zu ändern (Image MOD) 
kommt eine Fehler Meldung 
MFG

**Comment by kristian14192** [comment]
hi on my audi q7 MHI2_er_au37x_p3620 today I tried to install MIB and then GEM displayed this... can you advise why?
![454810551_8089167894501847_2260246907940596156_n](https://github.com/user-attachments/assets/e031ce93-4ebe-457e-98c3-165e9739e0fa)

**Comment by drakengard780** [comment]
Any idea for thise version Seat Arona 2021?
![17481640204103489354579808785782](https://github.com/user-attachments/assets/3e1ac5a4-f316-4dfe-a6ae-75cc2a6afe77)

**Comment by Dark251987** [comment]
![17659621778486165107172207123140](https://github.com/user-attachments/assets/6c50591c-7d9d-4e6b-8ff6-8569e065b882)
Hy. For this software CP off existing solutions?

**Comment by BooBoo124** [comment]
[https://mib.plain.gg/MHI2%20-%20HARMAN](url)

**Comment by Dragon7820** [comment]
I have a 6 c7 and errors from the previous owner.   what patch should I upload?
![Screenshot_20260320_105435_OBDeleven](https://github.com/user-attachments/assets/cbaddb1e-9fec-4fb4-8a1e-cfd31972775b)
![Screenshot_20260320_104852_OBDeleven](https://github.com/user-attachments/assets/bfe5768f-8adc-4460-9323-34ee300fedd8)

**Comment by v9m2tgpz4s-dot** [comment]
عند السيارةAudi Q7quattro2018 واردة أمريكي عم حاول تنزيل خريطة سوريا ما عم يمشي الحال
<img width="430" height="573" alt="image" src="https://github.com/user-attachments/assets/72d7e2a3-a373-4735-a12a-0f87fa478227" />

**Comment by v9m2tgpz4s-dot** [comment]
![Uploading IMG_9554.jpeg…]()
